### PR TITLE
feat: Qwen3-VL 배치 추론 지원 및 tps CLI 정합성 개선

### PR DESCRIPTION
**Pull request by beomsu-mobilint**
## 요약

- Qwen3-VL / Qwen2-VL 계열에 배치 추론 경로 추가 (`batch_size > 1` 지원, deepstack 포함).
- `mblt-model-zoo tps` CLI의 measure/sweep 출력과 워크플로우를 대대적으로 정리해 LLM/VLM · batch/non-batch 간 라벨·단위·순서 완전 일관화.
- 표 스키마를 데이터로 뽑아 공통 util(`mblt_model_zoo/cli/tps_table.py`)로 관리 — 향후 라벨/순서 변경은 스키마 한 자리에서 처리.
- 최근 리팩터로 어긋난 테스트 스텁 4건 재정합, pytest 워닝 42건 제거(SyntaxWarning + 의도된 slow-path 알림 노이즈).

## VLM 배치 추론

- `mblt_model_zoo/hf_transformers/models/qwen3_vl/modeling_qwen3_vl.py`, `qwen2_vl/modeling_qwen2_vl.py`, `utils/modeling_utils.py`에 배치 forward 경로 추가.
- `utils/npu_backend.py`: `target_cores` getter가 `target_clusters`를 확장하도록 개선, batched dispatch 전 `npu_time` reset.
- Qwen3-VL Batch16용 테스트 신설 (`tests/transformers/image_text_to_text/batch/`).
- 기존 VLM 테스트는 `image_text_to_text/non_batch/`로 재구성.

### 신규 지표
- `prefill_tps_per_w`, `decode_tps_per_w` (`tok/s/W`) — TPS/W 에너지 효율 지표를 measure/sweep 양쪽에 추가.
- `prefill_j_per_tok`, `decode_j_per_tok` (`J/tok`) — 토큰당 에너지.
- `prefill_energy`, `decode_energy`, `total_energy`, `vision_energy` — phase별 에너지 노출.

### 진행률 UX
- Warmup/measure 루프에 token-level tqdm 노출 (`show_progress=True`, `progress_prefix`).
- VLM warmup을 benchmark 스크립트(`_VLM_WARMUP_PREFILL=128`, `prefill_range=(128,128,128)`, `cache_lengths=[128]`)와 정합시켜 warmup 시간이 분 단위 → 초 단위로 단축.
- Config print와 warmup tqdm 라인 겹침 해소.

### Batch 자동 스케일링
- Batch 모델(`--batch-size > 1`)에서 `--prefill-range`/`--cache-lengths` 미명시 시 `_BATCH_SWEEP_LENGTH_SCALE=4`로 자동 축소 (`benchmark_text_generation_models._target_sweep_lengths()` 로직을 CLI에 이식).
- 자동 축소 시 한 줄 로그로 원본→축소 값 안내, 명시적 flag가 있으면 스킵 및 이유 로그.

### 표 라벨/단위/순서 통일
- 축약 통일: `utilization→util`, `temperature→temp`, `memory_used→mem_used`, `total_memory→total_mem`, `npu_latency→npu_lat`, sweep suffix `(last_point)→(last)`.
- 단위 통일: TPS/W를 `tok/s/W`로 통일.
- 접두사 규칙 확립:
  - **LLM-only 출력**: 접두사 없음 (`prefill_tps`, `avg_power` 등).
  - **VLM 출력**: LLM phase 행은 `llm_` (`llm_prefill_tps`, `llm_avg_power` 등), vision phase 행은 `vision_`, aggregate (`total`, `total_energy`, `total_mem`)는 접두사 없음.
- 섹션 순서 표준: Throughput → Latency → NPU lat → Spec decode → Power → Util → Temp → Memory → Energy → Efficiency. 각 섹션 내부는 overall → prefill → decode → vision.
- Missing 필드 채우기: VLM measure에 LLM/vision phase split 전부 노출, LLM measure에 `prefill_energy`/`decode_energy` 추가, sweep에 `total_npu_lat(last)` 추가, VLM sweep LLM subtable에 phase split 전부 노출.

### 공통 util화
- 신규 `mblt_model_zoo/cli/tps_table.py` — `TpsRow` 데이터클래스 + `TPS_TABLE_SPEC`으로 표 스키마를 선언적으로 정의.
- `emit_table(section, values_by_key)` 하나로 4개 표(`llm_measure`, `llm_sweep`, `vlm_measure`, `vlm_sweep_llm`, `vlm_sweep_vision`) 출력을 관리.
- 스키마 계약을 assert하는 유닛 테스트 신설 (`tests/transformers/cli_tps/test_tps_table_schema.py`).

## 테스트/워닝 정리

- 최근 helper 시그니처 확장으로 뒤처졌던 테스트 스텁 4건 재동기화:
  - `test_benchmark_asr_cli.py`의 `fake_measure_target`이 새 `trace_path` kwarg를 받도록 확장.
  - `test_vlm_routing_and_parser.py`의 stub `Args`에 `_auto_detect_vlm_task`가 요구하는 최소 필드(`task_explicit` 등) 부여.
  - `test_benchmark_transformers_cli.py`가 4-positional `_run_model(args, label, base, pipeline)` 신 시그니처로 호출.
- `pytest tests/transformers` 워닝 42건 → 0건:
  - `generation_utils.py`의 RST `\ s` 이스케이프가 트리거하던 `SyntaxWarning`을 raw docstring으로 정리.
  - last-only MXQ Path 3을 의도적으로 검증하는 fallback 테스트들의 `UserWarning`은 `pyproject.toml`의 `filterwarnings`에 message-prefix 매치로 억제(프로덕션 경고 자체는 사용자용 시그널이라 그대로 유지).
  - Llama cache 테스트가 slow-path에 빠지지 않도록 forward 호출에 `logits_to_keep=1`을 명시 (HF `.generate()`와 동일한 fast-path).

## Test plan
- [x] `mblt-model-zoo tps measure --model mobilint/Llama-3.2-1B-Instruct-Batch16`
- [x] `mblt-model-zoo tps measure --model mobilint/Qwen3-VL-8B-Instruct-Batch16`
- [x] `mblt-model-zoo tps sweep --model mobilint/Llama-3.2-1B-Instruct-Batch16`
- [x] `mblt-model-zoo tps sweep --model mobilint/Qwen3-VL-8B-Instruct-Batch16`
- [x] `pytest tests/transformers/cli_tps/`
- [x] `pytest tests/transformers/image_text_to_text/batch/`
- [x] `pytest tests/transformers/image_text_to_text/non_batch/`
- [x] `pytest tests/transformers/` — 771 passed, 1 skipped, 44 deselected, 0 warnings

### CODE REVIEW

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Comment by beomsu-mobilint** [comment]
@mobilint-review

File: mblt_model_zoo/cli/tps_table.py [hunk 1]
```diff
@@ -0,0 +1,998 @@
...
+        "decode_avg_power", "decode_avg_power", "W", _ALL_LLM, "decode_avg_power_w", llm_prefix=True
+    ),
+    _scalar_device_row(
+        "decode_p99_power", "decode_p99_power", "W", _ALL_LLM, "decode_p99_power_w", llm_prefix=True
+    ),
+    _scalar_device_row("vision_avg_power", "vision_avg_power", "W", _VLM_VISION, "vision_avg_power_w"),
+    _scalar_device_row("vision_p99_power", "vision_p99_power", "W", _VLM_VISION, "vision_p99_power_w"),
+
+    # --- Device metrics: utilization ---
+    _scalar_device_row("avg_util", "avg_util", "%", _ALL_LLM, "avg_utilization_pct", llm_prefix=True),
```
[the remark this answers is not in the document]
Comment (beomsu-mobilint) [reply mblt_model_zoo/cli/tps_table.py:612]: Intentional breaking change — won't fix.

The abbreviated schema labels (`avg_util`, `avg_temp`, `total_mem`, `npu_lat`, VLM `llm_avg_power`, `(last)` suffix) are the target CLI UX and cannot revert. The prior "CLI labels unchanged" line in the 2.3.0 changelog was a documentation oversight from the SoT-unification commit; a follow-up task is correcting the changelog to describe the CLI label migration explicitly (in addition to the JSON schema changes already listed).

The affected label changes originated in earlier commits in this PR chain (`05174a7`, `ffd111c`, `e60e702`, `b305fc7`) and are considered part of the 2.3.0 breaking change. `TpsRow` will keep a single `label` field serving both CLI and JSON; no separate display-label split is being introduced.

The 2.2.2 → 2.3.0 minor bump and the pending CHANGELOG update are the compatibility signal for downstream consumers.

**Comment by beomsu-mobilint** [comment]
@mobilint-review

**Review by parkjinman98 — APPROVED** [review]
